File: Day 5/Homework/Passing Data Property Segues ObjC/Passing Data Property Segues ObjC/ViewController2.m
//
//  ViewController2.m
//  Passing Data Property Segues ObjC
//
//  Created by Consultant on 8/24/22.
//

#import "ViewController2.h"

@interface ViewController2 ()
@property (weak, nonatomic) IBOutlet UILabel *myLabel;
//passTextSegue defined in header
@end

@implementation ViewController2

//var passTextSegue = ""
//@IBOutlet weak var myLabel: UILabel!
//override func viewDidLoad() {
//    myLabel.text = passTextSegue
//    super.viewDidLoad()
//}



- (void)viewDidLoad {
    
    [super viewDidLoad];
    _myLabel.text = _passTextSegue;
    // Do any additional setup after loading the view.
}

/*
#pragma mark - Navigation

// In a storyboard-based application, you will often want to do a little preparation before navigation
- (void)prepareForSegue:(UIStoryboardSegue *)segue sender:(id)sender {
    // Get the new view controller using [segue destinationViewController].
    // Pass the selected object to the new view controller.
}
*/

@end

File: Day 4/NavigationExample/NavigationExample/ViewController2.swift
//
//  ViewController2.swift
//  NavigationExample
//
//  Created by Consultant on 8/23/22.
//

import UIKit

class ViewController2: UIViewController {

    var segue2 = "mySegue2"
    
    override func viewDidLoad() {
        super.viewDidLoad()

        // Do any additional setup after loading the view.
    }
    
    @IBAction func To3(_ sender: Any) {
        performSegue(withIdentifier: segue2, sender: nil)
    }
    
    /*
    // MARK: - Navigation

    // In a storyboard-based application, you will often want to do a little preparation before navigation
    override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
        // Get the new view controller using segue.destination.
        // Pass the selected object to the new view controller.
    }
    */

}
